### PR TITLE
Story security

### PR DESCRIPTION
**Pull request by katestarks**
username: katestarks
password: SecurePassword

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -27,6 +27,7 @@
         <div><a href="#about"><img class="heroArrow" src="IMG/heroArrow.png"></a></div>
     </main>
     <nav>
+        <a href="login_page.php"><img class="loginIcon" src="IMG/loginIcon.jpg"></a>
```
Comment (charliecog) [inline index.php]: alt text

File: login_page.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <link href="https://fonts.googleapis.com/css?family=Quicksand:300,500" rel="stylesheet">
+    <link rel='stylesheet' type='text/css' href='css/cms_styles.css'/>
+</head>
+<body>
+<h1>Log in to Manage Content for Kate's Portfolio Page</h1>
+    <form method="POST" action="cms_index.php">
+        <label for="username"><h4>Username:</h4></label>
+        <input type = "text" class="typeText" name="username">
+        <label for="password"><h4>Password:</h4></label>
+        <input type = "text" class="typeText" name="password">
```
Comment (charliecog) [inline login_page.php]: input type should be password

Comment (charliecog) [reply login_page.php]: and remove spaces between type and = and text

File: login_page.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <title>Log in to Kate's Portfolio CMS</title>
+    <link rel='stylesheet' type='text/css' href='css/normalize.css'/>
+    <link href="https://fonts.googleapis.com/css?family=Quicksand:300,500" rel="stylesheet">
+    <link rel='stylesheet' type='text/css' href='css/cms_styles.css'/>
+</head>
+<body>
+<h1>Log in to Manage Content for Kate's Portfolio Page</h1>
+    <form method="POST" action="cms_index.php">
+        <label for="username"><h4>Username:</h4></label>
+        <input type = "text" class="typeText" name="username">
```
Comment (charliecog) [inline login_page.php]: remove spaces between type and = and text

File: php/functions.php [hunk 4]
```diff
@@ -241,4 +241,11 @@ function deleteAboutMeSuccess(bool $aboutMeTextDeleted) : string {
         return "<p>The paragraph has been deleted</p>";
     }
 }
+
+function fetchCredentials(PDO $db) : array {
```
Comment (charliecog) [inline php/functions.php]: doc block?

**Review by charliecog — COMMENTED** [review]
looks good, couple of things but almost there

File: index.php (hunk 1 above)
Comment (katestarks) [reply index.php]: done

File: login_page.php (hunk 2 above)
Comment (katestarks) [reply login_page.php]: done

File: login_page.php (hunk 3 above)
Comment (katestarks) [reply login_page.php]: done

File: php/functions.php (hunk 4 above)
Comment (katestarks) [inline php/functions.php:252]: done

**Review by charliecog — APPROVED** [review]
woop woop!

**Pull request merged**